completions.go

```diff
@@ -899,3 +900,35 @@ func CompError(msg string) {
 func CompErrorln(msg string) {
 	CompError(fmt.Sprintf("%s\n", msg))
 }
+
+// These values should not be changed: users will be using them explicitly.
+const (
+	configEnvVarGlobalPrefix       = "COBRA"
+	configEnvVarSuffixDescriptions = "COMPLETION_DESCRIPTIONS"
+	configEnvVarDescriptionsOff    = "off"
```

Comment:
Those suggestions make sense to me, I'm not that hung up on `on/off`.

One issue of starting to accept true values for this is that setting a true value won't make e.g. descriptions magically appear if they haven't been written, whereas a false value will definitely turn then off no matter if they're implemented or not. But IMO that's a tiny one that can be addressed by wording related docs appropriately.

I'd keep the full `COMPLETION_DESCRIPTIONS` for clarity and not abbreviate it. Although `_ON` would be good readabilitywise, it would be inconsistent with `ACTIVE_HELP`.

Reply:
Also, `strconv.ParseBool` returns an error if the value is not one of the acceptable ones. We could handle it for descriptions, but if we want to make `ACTIVE_HELP` behave the same, there is a backwards incompatibility chance.

Reply:
138b0ab96495545094a0576d989241872687f886